sklearn/tree/plot_iforest.py

```diff
@@ -0,0 +1,54 @@
+"""
+==========================================
+IForest example
+==========================================
+
+An example using IForest for anomaly detection.
```

Comment:
It would be great to have a few lines of explanation of the `IForest` and also a few lines explaining this particular example too.. ( also, if its not too much to ask for, perhaps a few lines on fine tuning the parameters (the `psi` / `n_tree`?) perhaps... )